### PR TITLE
No Grid Snap Keyboard Shortcut

### PR DESCRIPTION
**Pull request by kdparkinson**
Press alt while moving an object with grid on and the item will not be forced to snap to the grid.

### CODE REVIEW

**Comment by 4ian** [comment]
Thanks for this! I guess it will be useful :) 
Seems like you've selected the wrong branch when creating the pull request: 
image

This should be `4ian:master` (hence all the commits displayed in the PR). I'll delete my `new-ide` to avoid confusion in the future :)

**Comment by 4ian** [comment]
I've changed the base to `4ian:master` (using the Edit button) for you ;) 
This looks good! I think it's safe to merge :)

File: newIDE/app/src/InstancesEditor/InstancesMover.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export default class InstancesResizer {
+export default class InstancesMover {
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/InstancesMover.js]: Thanks for doing this :)

File: newIDE/app/src/InstancesEditor/InstancesMover.js [hunk 2]
```diff
@@ -11,28 +11,24 @@ export default class InstancesResizer {
...
   }
 
-  _roundYPosition(y) {
-    if (
-      !this.options.snap ||
-      !this.options.grid ||
-      this.options.gridHeight <= 0
-    )
+  _roundYPosition(y, noGridSnap) {
+    if (!this.options.snap || !this.options.grid || this.options.gridHeight <= 0 || noGridSnap)
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/InstancesMover.js]: Can you run prettier on the files? In VSCode, once you have prettier extension installed, press Cmd+Shift+P and enter "Format Document".

Prettier is an automatic code formatter, all the code is formatted by this :)

Comment (kdparkinson) [reply newIDE/app/src/InstancesEditor/InstancesMover.js]: Will do.

**Comment by 4ian** [comment]
Awesome! This looks great, thanks for adding this!

**Comment by blurymind** [comment]
Nice addition! Thank you  :) 👍

**Pull request merged**